scripts/validate_docstrings.py

```diff
@@ -531,6 +537,13 @@ def validate_one(func_name):
         examples_errs = doc.examples_errors
         if examples_errs:
             errs.append('Examples do not pass tests')
+        examples_source_code = doc.examples_source_code
```

Comment:
may be you can do the join here, so doesn't need to be repeated? also, is the lines end with `\n` I'd join by `''`, otherwise by `'\n'`. Doesn't change the result, but conceptually the content of the variable is something that makes more sense.